doc/kube-api-test.md

```diff
@@ -0,0 +1,153 @@
+# kube-api-test
+
+`kube-api-test` makes it easy to implement integration tests with Kubernetes API Server in Java.
+Inspired by [envtest](https://book.kubebuilder.io/reference/envtest.html) in Kubebuilder (Golang).
+
+It runs the API Server binaries directly (without nodes and other components, but with etcd).
+Linux, Windows, Mac is supported.
+
+## Usage
+
+Include dependency:
+
+```xml
+<dependency>
+    <groupId>io.fabric8</groupId>
+    <artifactId>kube-api-test</artifactId>
+    <version>[version]</version>
+    <scope>test</scope>
+</dependency>
+```
+
+### In Unit Tests
+
+See sample unit
+test [here](https://github.com/csviri/kubernetes-client/blob/main/junit/kube-api-test/core/src/test/java/io/fabric8/kubeapitest/sample/JUnitExtensionSimpleCaseTest.java)
+
+```java
+
+@EnableKubeAPIServer
+class JUnitExtensionSimpleCaseTest {
+
+    // Use @KubeConfig annotation to inject kube config yaml to init any client
+    @KubeConfig
+    static String kubeConfigYaml;
+
+    @Test
+    void simpleTestWithTargetVersion() {
+        var client = new KubernetesClientBuilder()
+                .withConfig(Config.fromKubeconfig(kubeConfigYaml))
+                .build();
+
+        client.resource(TestUtils.testConfigMap()).create();
+        var cm = client.resource(TestUtils.testConfigMap()).get();
+
+        Assertions.assertThat(cm).isNotNull();
+    }
+}
+```
+
+### Public API
+
+The underlying API can be used directly.
+See [KubeApiServer](https://github.com/csviri/kubernetes-client/blob/main/junit/kube-api-test/core/src/main/java/io/fabric8/kubeapitest/KubeAPIServer.java#L28-L28)
```

Comment:
I think these links need to be updated to be under `fabric8io` organization

Reply:
updated, thx!